app/templating/summary/question.py

```diff
@@ -67,11 +80,11 @@ def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
         for option in answer_schema['options']:
             if option['value'] in answer:
-                if option['value'] == 'other':
+                if option['value'].lower() == 'other':
                     summary_option_display_value = cls._get_checkbox_other_display_value(answer, answer_schema, option)
-                    multiple_answers.append(summary_option_display_value)
+                    multiple_answers.append((summary_option_display_value, True))
```

Comment:
The Boolean values indicate if this option is the "other" option.  If you look at the surrounding `if` statement then really those boolean values are equivalent to `option['value'].lower() == 'other'` but I agree, it's not immediately obvious. Perhaps this should be refactored by extracting a variable?